src/app/lib/config/toggles/const.js

```diff
@@ -2,7 +2,6 @@ export default {
   ADS: 'ads',
   CHARTBEAT_ANALYTICS: 'chartbeatAnalytics',
   COMSCORE_ANALYTICS: 'comscoreAnalytics',
-  CPS_RECOMMENDATIONS: 'cpsRecommendations',
```

Comment:
Can we also delete this toggle from test & live iSite once the PR is merged?

Reply:
Yea we can 👍